### DISCUSSION TITLE
About editing metadata "Restricting Direct Downloads" from LVFS UI web

### DISCUSSION THREAD

**Discussion by vincent-tang1** [fwupd/fwupd#6968] [Q&A] [answered]
**Describe the question**
Just a small question, could we set "Restricting Direct Downloads" condition by modifying online metadata from LVFS UI web ?
Or, can we only add it before upload CAB file ? Thanks!!
```
<!-- most OEMs do not need to do this... -->
<custom>
  <value key="LVFS::InhibitDownload"/>
</custom>
```

**Comment by hughsie** [comment, marked as answer]
> can we only add it before upload CAB file

I think it's only available pre-upload at the moment. I guess we could add UI for it -- but it's only used by one other vendor IIRC.

**Comment by vincent-tang1** [comment]
Got it, thanks!
